sql-cloudant/src/main/scala/org/apache/bahir/cloudant/common/JsonStoreRDD.scala

```diff
@@ -101,24 +105,24 @@ class JsonStoreRDD(sc: SparkContext, config: CloudantConfig)
       case LessThanOrEqual(attr, v) => ("$lte", v)
       case _ => (null, null)
     }
-    val convertedV: JsValue = {
+    val convertedV: JsonElement = {
       // TODO Better handing of other types
       if (value != null) {
         value match {
-          case s: String => Json.toJson(s)
-          case l: Long => Json.toJson(l)
-          case d: Double => Json.toJson(d)
-          case i: Int => Json.toJson(i)
-          case b: Boolean => Json.toJson(b)
-          case t: java.sql.Timestamp => Json.toJson(t)
+          case s: String => parser.parse(s)// Json.toJson(s)
```

Comment:
some old code here that should probably be removed (and on the following line)

Reply:
Removed in 55cc844.